Dockerfile

```diff
@@ -30,7 +30,7 @@ RUN apk add --no-cache --initdb --root /out \
     openssh \
     && true
 
-ENV KUBECTL_VERSION v1.11.5
+ENV KUBECTL_VERSION v1.15.11
```

Comment:
@sayboras  was the older version causing issues?

Reply:
oh it's not causing any issue. 

As the process of updating dockerfile from contributor might be a little bit tedious, so I just take this change to make it same as default EKS version at that point of time.